packages/design-system/src/connect/components/document-toolbar-v2/utils/use-document-timeline.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  useSelectedDocument,
+  useTimelineItems,
+} from "@powerhousedao/reactor-browser";
+
+export function useDocumentTimeline() {
```

Comment:
@acaldas @gpuente 

same for here, if this is only intended to be used by the document toolbar then it might as well just live in that component. otherwise we should export it from reactor browser so that we and others can also use it